com.github.chipmuenk.pyfda.yaml

```
@@ -0,0 +1,58 @@
+app-id: com.github.chipmuenk.pyFDA
+runtime: org.kde.Platform
+runtime-version: '5.12'
+sdk: org.kde.Sdk
+command: pyfdax
+finish-args:
+- "--socket=wayland"
+- "--socket=x11"
```

```suggestion
- "--socket=fallback-x11"
```

assuming proper wayland support, otherwise drop the wayland socket.